### ISSUE TITLE
0.8.1.2230 is not working with PotPlayer

### ISSUE THREAD

**Issue by yezy98** [Aleksoid1978/VideoRenderer#155]
When I was using 0.7.3.2210, everything works well. However, after I update to the latest release, the RTX Super Resolution and HDR just don't work although I turned them on in the settings.

I always use PotPlayer, and after this issue happened, I tried the MPC-BE.
This also happens on MPC-BE.
Then I uninstall the MPC VR and installed 0.7.3.2210.
And it works well on both PotPlayer and MPC-BE

I am using Windows Canary Insider Build 26200.5001 and my graphic driver is Geforce Game Ready 552.22

**Comment by v0lt** [comment]
The current version of MPC Video Renderer is 0.8.1.2230.
Most recently there were versions 0.8.0.x. You can find [it there](https://github.com/qwerttvv/Player/releases).

Windows 7-11 are supported to varying degrees. Insider Windows are not supported.

**Comment by Aleksoid1978** [comment]
Show MPC VR settings, show Nvidia settings, show screenshot with statistics in MPC VR.

**Comment by yezy98** [comment]
I just tried again to get some screenshots of the info you requested and found that actually MPC-VR 0.8.1.2230 works well with MPC-BE 1.7.1.
Maybe I just didn't check everything carefully yesterday.

However, I found that on PotPlayer, even the statistics is not shown.
And from the 'Playback/System Information', I found that the filter is not used.
![render 0 8 1](https://github.com/Aleksoid1978/VideoRenderer/assets/65610192/bcea0e2d-5cad-4c21-a8a6-278ca235b610)
![Used Filter 0 8 1](https://github.com/Aleksoid1978/VideoRenderer/assets/65610192/c2cec721-504c-4cb4-8511-a3dbbf50db0d)

I think that means the MPC-VR is not running.

I also tried MPC-VR 0.8.0-2229 on PotPlayer, and it doesn't work as well.
